infra.bs

```diff
@@ -985,7 +985,10 @@ literal, we can use plainer language: <var>domain</var> <a for=string>ends with<
 {{Array/sort()}} method on an array of strings. This ordering compares the 16-bit code units in each
 string, producing a highly efficient, consistent, and deterministic sort order. The resulting
 ordering will not match any particular alphabet or lexicographic order, particularly for
-<a>code points</a> represented by a surrogate pair. [[!ECMA-262]]
+<a>code points</a> represented by a surrogate pair. For example, the codepoint U+FF5E
+(～ FULLWIDTH TILDE) is obviously less than the codepoint U+1F600 (😀), but the tilde is
```

Comment:
"codepoint" -> "code point" in two places

Per https://infra.spec.whatwg.org/#code-points the exact notation should be `U+FF5E FULLWIDTH TILDE (～)`